ui-pf4/pom.xml

```diff
@@ -120,7 +120,19 @@
             <artifactId>graphene-webdriver</artifactId>
             <type>pom</type>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>net.sourceforge.htmlunit</groupId>
+                    <artifactId>htmlunit</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
+        <dependency>
+            <groupId>net.sourceforge.htmlunit</groupId>
+            <artifactId>htmlunit</artifactId>
+            <version>2.70.0</version>
```

Comment:
Worth adding the scope test

Reply:
Definitely YES. I've just added the test scope in my last commit